### PR TITLE
Success stories

### PR DESCRIPTION
**Pull request by Jood80**
the basic elements of this component has been added, pagination is in progress as well as Back end part

### CODE REVIEW

**Comment by Jood80** [comment]
the connection between Back & Front has been accomplished successfully ! :tada:

File: client/src/App.js [hunk 1]
```diff
@@ -17,6 +16,7 @@ function App() {
 			<Router>
 				<Switch>
 					<Route path="/" exact component={Home} />
+					<Route path= "/api/home/success-stories" exact component={Home} />
```
Comment (RandInaim) [inline client/src/App.js]: what is this!!

**Review by RandInaim — CHANGES_REQUESTED** [review]
please remove unnecessary files! 
please make sure of your eslint and prettier configurations

is this your final output?!
image

File: client/src/App.js (hunk 1 above)
Comment (Jood80) [reply client/src/App.js]: ah don't worry, I find this after pushing the request so I updated it but didn't updated her yet.

**Comment by Jood80** [comment]
@RandInaim  The current design is not responsive yet, I make the priority to the back end at this point so I pause the work on it, in order to come back while finishing process. However it looks fine for the screens of "x1273 width " right now.
image

**Comment by RandInaim** [comment]
slide show is very fast and farah's photo is not showing up

**Comment by Jood80** [comment]
@RandInaim it has been fixed :v:

**Comment by RandInaim** [comment]
other comments need to be solved in refactor weeks

**Pull request merged**